frontend/src/App.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import BotsList from "./BotsList";
+import Navbar from "./Navbar";
+import categoryDetail from "./categoryDetail";
```

Comment:
Components sould ALWAYS be named starting with capital
```suggestion
import CategoryDetail from "./CategoryDetail";
```